### PR TITLE
Beato d10 7 8

### PR DESCRIPTION
**Pull request by Avuxo**
these files were actually pretty damn good, not much to TLC

### CODE REVIEW

File: script/Arcueid/Day 10/10_00_ARC10_7.txt [hunk 1]
```diff
@@ -16,12 +16,13 @@ Besides, you couldn't kill that kid with your skill. You'd be the one killed ins
 [sha:12c35238b462c193b2ac60577440ba9fbb8b9ce7]{
 -- Page 756, Offset 26324.
 -- 「なんでって、そろそろ<調律|ピアノ>の影響が出る頃だからだよ。
+// double check: this is correct literally, but I'm not sure what the fuck it means
```
Comment (nolo16) [inline script/Arcueid/Day 10/10_00_ARC10_7.txt:19]: I'm pretty sure this is right (this is referring to Mario's power I think?) iirc its called Piano Machine or something like that

File: script/Arcueid/Day 10/10_00_ARC10_7_2B.txt [hunk 2]
```diff
@@ -21,7 +21,7 @@ If that's so―――if I search for a place without lines while looking over th
 [sha:d097092e5c1e16d6205645da4b92cc330c34c54f]{
 -- Page 775, Offset 26390.
 -- 短絡的だが、これが今の俺に出来る唯一の手段だ。
-It's far too simplistic, but this is the only means I have as of now.
+This feels like a rudimentary strategy, but it's all of got.
```
Comment (nolo16) [inline script/Arcueid/Day 10/10_00_ARC10_7_2B.txt:24]: I've got

**Review by nolo16 — APPROVED** [review]
otsucurry

File: script/Arcueid/Day 10/10_00_ARC10_7_2B.txt (hunk 2 above)
Comment (Avuxo) [reply script/Arcueid/Day 10/10_00_ARC10_7_2B.txt]: these typos are always the most embarrassing. Like how did my wires get so crossed that I thought "of" was "I've"

**Pull request merged**